Code/Tools/AssetProcessor/native/AssetManager/SourceFileRelocator.h

```diff
@@ -76,6 +76,15 @@ namespace AssetProcessor
         Succeeded
     };
 
+    enum RelocationParameters
+    {
+        PreviewOnlyFlag = 1 << 0,
+        RemoveEmptyFoldersFlag = 1 << 1,
+        AllowDependencyBreakingFlag = 1 << 2,
+        UpdateReferencesFlag = 1 << 3,
+        ExcludeMetaDataFilesFlag = 1 << 4
```

Comment:
update: As this is a plain enum please prefix the values with `RelocationParameters_` so they can't as easily get confused with other enums

Reply:
Done.